randovania/games/game.py

```diff
@@ -32,6 +32,9 @@
     from randovania.layout.preset_describer import GamePresetDescriber
     from randovania.resolver.bootstrap import Bootstrap
 
+# Type alias for progressive items
+Progressive_item_tuples = list[tuple[str, tuple[str, ...]]]
```

Comment:
Iirc we use PascalCase so far for a type alias. Like `TeleporterConnection` etc.

Reply:
Technically it should be `Iterable[tuple[str, tuple[str, ...]]]`, and please also update GameGui.progressive_item_gui_tuples :D